File: 15820.cpp
#include <iostream>
using namespace std;

#define CORRECT 0
#define WRONG_SAMPLE 1
#define WRONG_SYSTEM 2

int main() {
	int S1, S2; cin >> S1 >> S2;

	int result = CORRECT;
	for (int i = 0; i < S1; i++) {
		long long a, b; cin >> a >> b;
		if (a != b) result = WRONG_SAMPLE;
	}

	if (!result) {
		for (int i = 0; i < S2; i++) {
			long long a, b; cin >> a >> b;
			if (a != b) result = WRONG_SYSTEM;
		}
	}

	if (result == CORRECT)
		cout << "Accepted\n";
	else if (result == WRONG_SAMPLE)
		cout << "Wrong Answer\n";
	else
		cout << "Why Wrong!!!\n";
}

File: 15820.py
CORRECT = 0; WRONG_SAMPLE = 1; WRONG_SYSTEM = 2
S1, S2 = map(int, input().split())

result = CORRECT
for i in range(S1):
    a, b = map(int, input().split())
    if a != b: result = WRONG_SAMPLE

if result == CORRECT:
    for i in range(S2):
        a, b = map(int, input().split())
        if a != b: result = WRONG_SYSTEM

if result == CORRECT:
    print("Accepted")
elif result == WRONG_SAMPLE:
    print("Wrong Answer")
else:
    print("Why Wrong!!!")
